test/Streams.purs

```diff
@@ -1,24 +1,20 @@
 module Test.Streams where
 
-import Prelude
-import Data.Maybe
-import Data.Either
+import Prelude (Unit, bind, (==))
```

Comment:
If we make this open, can we get rid of the imports of Control.Apply and Control.Bind too?